packages/corvid-cli/src/index.debug.js

```diff
@@ -1,9 +1,19 @@
 #!/usr/bin/env node
-process.env.NODE_ENV = process.env.NODE_ENV || "development";
-process.env.LOG_LEVEL = process.env.LOG_LEVEL || "debug";
-process.env.CORVID_CLI_DISABLE_HEADLESS = true;
-process.env.CORVID_CLI_DEVTOOLS = true;
-process.env.FORCE_COLOR = true;
+const chalk = require("chalk");
+const version = require("./version");
```

Comment:
you should check the version before doing anything else (here for example, you require "chalk" and "./messages" before.

i would just create a "checkNodeVersion.js" file that throws and exists the process if the node version isn't valid. no need to export a "checkVersion" function

Reply:
I need chalk and messages to show a message